diff -Naur mozilla/browser/app/profile/firefox.js mozilla.tpg/browser/app/profile/firefox.js
--- mozilla/browser/app/profile/firefox.js	2008-02-04 19:36:22.000000000 +0000
+++ mozilla.tpg/browser/app/profile/firefox.js	2008-02-29 21:26:08.000000000 +0000
@@ -184,7 +184,7 @@
 
 // Whether or not the application should check at startup each time if it 
 // is the default browser.
-pref("browser.shell.checkDefaultBrowser", true);
+pref("browser.shell.checkDefaultBrowser", false);
 
 // 0 = blank, 1 = home (browser.startup.homepage), 2 = last visited page, 3 = resume previous browser session
 // The behavior of option 3 is detailed at: http://wiki.mozilla.org/Session_Restore
